test/conf_include_test.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2016-2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <openssl/conf.h>
+#include <openssl/err.h>
+#include "testutil.h"
+
+#ifdef _WIN32
+# include <direct.h>
+# define DIRSEP "/\\"
+# define chdir _chdir
+#else
+# include <unistd.h>
+# define DIRSEP "/"
```

Comment:
Note that on VMS, the native directory spec looks like this: `vol:[dir1.dir2]`, where the `]` must be preserved to form a valid directory.  It may *also* simply be `foo:`, i.e. just a device name (and yes, the colon must be preserved).  However, C programs *also* understand the Unix form, which should be handled accordingly.

Who said life was simple? ;-)

Reply:
OK, I've tried to accommodate that.